src/recon_buildblock/distributable.cxx

```diff
@@ -435,12 +435,10 @@ void distributable_computation(
 #pragma omp for schedule(runtime)  
 #endif
 
-  for (int timing_pos_num = min_timing_pos_num; timing_pos_num <= max_timing_pos_num; ++timing_pos_num)
-  {
     // note: older versions of openmp need an int as loop
     for (int i=0; i<static_cast<int>(vs_nums_to_process.size()); ++i)
       {
```

Comment:
In terms of OpenMP, this code block meant that only `timing_pos_num`s were threaded.